access-control-app/build-access-control-for-container.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+lein uberjar
+docker build -t access-control-app .
+docker run access-control-app
```

Comment:
This is a build and **run** script. When would we use these shell scripts? I think some documentation in the README files for starting up the application using Docker is probably good enough for now.